components/inputs/test/input-checkbox.vdiff.js

```diff
@@ -0,0 +1,108 @@
+import '../input-checkbox.js';
+import '../input-checkbox-spacer.js';
+import { expect, fixture, focusElem, html } from '@brightspace-ui/testing';
+import { loadSass, unloadSass } from '../../../test/load-sass.js';
+
+describe('d2l-input-checkbox', () => {
+
+	before(loadSass);
+	after(unloadSass);
+
+	[false, true].forEach(skeleton => {
+		[false, true].forEach(disabled => {
+			const checkedStates = ['checked', 'unchecked'];
+			if (!skeleton) {
+				checkedStates.push('indeterminate');
```

Comment:
The old HTML file actually has fixtures for skeleton + indeterminate but I've excluded them to be consistent. I can add them in though if we think they're valuable.

Reply:
Interesting, as in had fixtures for them but no actual tests/goldens? It would clean up three lines of code for some (hopefully) simple, non-flaky diffs, but not sure how _valuable_ they are.  I'd say I'm indifferent

Reply:
Yeah exactly. They're not testing anything interesting IMO so I'm going to leave them off.